pkg/local_object_storage/metabase/status.go

```diff
@@ -64,7 +64,8 @@ func (db *DB) ObjectStatus(address oid.Address) (ObjectStatus, error) {
 
 		removedStatus := inGraveyardWithKey(addrKey, graveyardBkt, garbageObjectsBkt, garbageContainersBkt)
 
-		if removedStatus != 0 && objectLocked(tx, cID, oID) || inBucket(tx, primaryBucketName(cID, key), objKey) || inBucket(tx, parentBucketName(cID, key), objKey) {
+		childForParent := getChildForParent(tx, cID, oID, key)
```

Comment:
i'd inline, it may be not needed

Reply:
Currently it's used twice.

Reply:
i mean var, not func

Reply:
That condition (the next line) is complex enough already.

Reply:
This method is far from being optimal DB-wise (doube `objectLocked`, a lot of the same accesses in `readBuckets`), but it's only used for control which is not on the hot path.